src/globals/scss/_theme.scss

```diff
@@ -158,8 +158,10 @@
 
   // Accordion (Reverse)
   $accordion-flex-direction: row !default !global;
+  $accordion-flex-direction-2: row-reverse !default !global;
   $accordion-justify-content: flex-start !default !global;
   $accordion-arrow-margin: 0 0 0 $spacing-2xs !default !global;
+  $accordion-arrow-margin-2: 0 $spacing-2xs 0 0 !default !global;
```

Comment:
Can you explain whats going on with the `-2` variables? Are both being used? Can we use a different name?

Reply:
the `arrow-margin` and `flex-direction` variables are both being used, so wasn't to sure what to name them...definitely can change the name of these.

Reply:
Where are they being used? I don't see them in the experimental accordion at all? You can update anything in the experimental theme mixin to what you need for the experimental components. It won't affect the default since that uses the default theme.